### PR TITLE
Updated the readme file of shopify inetgration

### LINKED ISSUE #8434: [Bug]: Shopify handler cannot connect when providing Yotpo keys
### Short description of current behavior

Shopify handler: https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/shopify_handler

When connecting Shopify to MindsDB without Yotpo app, it works well:

image

But when connecting it with Yotpo access key (generated using [this instruction](https://support.yotpo.com/docs/finding-your-yotpo-app-key-and-secret-key)), then it cannot connect:

image

### Video or screenshots

_No response_

### Expected behavior

Shopify should connect to MindsDB with Yotpo keys that allow for querying customer reviews.

### How to reproduce the error

_No response_

### Anything else?

_No response_

### LINKED ISSUE #8436: [Docs]: Add a note to shopify integration readme file for  generating yotopo access token as some people can get confused between yotop access token and secret key
### Short description of what should be added or improved

we have to add a note to generate yotopo access token as some people can get confused between secret key and access token

### Video or screenshots

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Sekhar-Kumar-Dash**
Fixes #8434 

## Type of change
- [x] 📄 This change requires a documentation update

## Verification Process

To ensure the changes are working as expected:

 - [ ]   Test Location: Specify the URL or path for testing.
 - [ ]   Verification Steps: Outline the steps or queries needed to validate the change. Include any data, configurations, or actions required to reproduce or see the new functionality.

## Additional Media:

- [ ] I have attached a brief loom video or screenshots showcasing the new functionality or change.

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have appropriately commented on my code, especially in complex areas.
- [x] Necessary documentation updates are either made or tracked in issues.
- [x] Relevant unit and integration tests are updated or added.

### CODE REVIEW

File: mindsdb/integrations/handlers/shopify_handler/README.md [hunk 1]
```diff
@@ -39,8 +39,8 @@ These are the optional parameters:
 - `yotpo_app_key`: token needed to access customer reviews via the Yotpo Product Reviews app.
 - `yotpo_access_token`: token needed to access customer reviews via the Yotpo Product Reviews app.
 
-If you want to query customer reviews, use the [Yotpo Product Reviews](https://apps.shopify.com/yotpo-social-reviews) app available in Shopify. To generate `yotpo_app_key` and `yotpo_access_token` keys, refer to [this](https://support.yotpo.com/docs/finding-your-yotpo-app-key-and-secret-key) and [this](https://apidocs.yotpo.com/reference/yotpo-authentication).
-
+If you want to query customer reviews, use the [Yotpo Product Reviews](https://apps.shopify.com/yotpo-social-reviews) app available in Shopify. To generate `yotpo_app_key` and `yotpo_access_token` keys, refer to [this](https://support.yotpo.com/docs/finding-your-yotpo-app-key-and-secret-key) and [this](https://apidocs.yotpo.com/reference/yotpo-authentication)(Note:please don't enter the app secret key instead of yotpo access token both are different).
+Note:please don't enter the app secret key instead of yotpo access token both are different
```
Comment (martyna-mindsdb) [inline mindsdb/integrations/handlers/shopify_handler/README.md:43]: Please replace lines 42 and 43 with the below content:
```
If you want to query customer reviews, use the [Yotpo Product Reviews](https://apps.shopify.com/yotpo-social-reviews) app available in Shopify. Here are the steps to follow:
1. Install the [Yotpo Product Reviews](https://apps.shopify.com/yotpo-social-reviews) app for your Shopify store.
2. Generate `yotpo_app_key` following [this instruction](https://support.yotpo.com/docs/finding-your-yotpo-app-key-and-secret-key) for retrieving your app key. Learn more about [Yotpo authentication here](https://apidocs.yotpo.com/reference/yotpo-authentication).
3. Generate `yotpo_access_token` following [this instruction](https://develop.yotpo.com/reference/generate-a-token).
```

**Comment by martyna-mindsdb** [comment]
@Sekhar-Kumar-Dash 
Please see the review comment above.

Also, please update this info in this file `docs/integrations/app-integrations/shopify` at line 29.

**Comment by Sekhar-Kumar-Dash** [comment]
In this PR:https://github.com/mindsdb/mindsdb/pull/8442                                                                                                           I have updated the shopify.mdx

**Comment by martyna-mindsdb** [comment]
Thanks @Sekhar-Kumar-Dash 

You don't need to create another PR. In future, include it in the same PR.

**Pull request merged**